src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/Program.cs

```diff
@@ -169,17 +212,8 @@ private static IProjectContext GetProjectInformation(string projectPath)
                 return new MsBuildProjectContextBuilder(projectFileFinder.ProjectFilePath, codeGenerationTargetsLocation)
                     .Build();
             }
-            else
-            {
-                return new DotNetProjectContextBuilder(
-                    projectFileFinder.ProjectFilePath,
-                    new NuGetFramework[] 
-                    {
-                        FrameworkConstants.CommonFrameworks.NetCoreApp10,
-                        FrameworkConstants.CommonFrameworks.Net451
-                    })
-                    .Build();
-            }
+
+            throw new InvalidOperationException($"Please provide a valid Project file. {projectFileFinder.ProjectFilePath}");
```

Comment:
Nit: suggest working `'{projectFileFinder.ProjectFilePath}' is not a valid MSBuild project file.`